### ISSUE TITLE
JSTL SQL with in query

### ISSUE THREAD

**Issue by d3ep4k** [metamug/mason#62]
https://stackoverflow.com/questions/2298550/oracle-any-vs-in

```sql
SELECT * from movie where name = ANY(?)
```
The above doesnt work in mysql.

Can these be used?
https://stackoverflow.com/questions/178479/preparedstatement-in-clause-alternatives

**Comment by kaisteel** [comment]
Yes, we can use one of the techniques given [here](https://stackoverflow.com/questions/178479/preparedstatement-in-clause-alternatives) but our problem is with the notation. Let me take your example to explain.
`SELECT * from movie where name = ANY(?)`
If we use `ANY(?)` this notation and pass value to this prepared statement say [2,3,4,5] then this is going to look like
`SELECT * from movie where name = ANY("2,3,4,5")`
where as in fact we want it to like this
`SELECT * from movie where name = ANY(2,3,4,5)`
For which the notation should be
`SELECT * from movie where name = ANY(?,?,?,?)`

**Comment by kaisteel** [comment]
So we'll first have to decide how many params (?) we will have to generate
`ANY(?,?,?....n)`
Once that is decided after that it's straight forward.
